app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,23 @@
+class CommentsController < ApplicationController
+  def new
+    @post = @current_user.posts.find(params[:post_id])
+  end
```

Comment:
- [ ] It will be great to initialize a new comment in your `new` method. What you are doing is getting the post. In Rails the `new` method is for creating an instance of what you want to create. What you are doing in your implementation should be done in the` show` method not in the new method. Hint ⬇️ 
`@comment = Comment.new`